rotkehlchen/exchanges/data_structures.py

```diff
@@ -139,6 +140,9 @@ class Trade(NamedTuple):
     # the exchanges themselves then there is no way to avoid duplicates.
     link: str
     notes: str = ''
+    # NB: custom_identifier is set for trades that store in the ID the base and
+    # quote assets data, e.g. AMMTrade (see `identifier` property below).
+    custom_identifier: TradeID = TradeID('')
```

Comment:
@LefterisJP this new field is a must if I want to add and read trades from DB that store base_asset and quote_asset data (both can be unknown tokens) and keeping the current trades DB logic. You will see how I use it around.